packages/devtools_app/lib/src/screens/provider/instance_viewer/result.dart

```diff
@@ -59,15 +57,13 @@ abstract class Result<T> with _$Result<T> {
       data: (value) => value,
       error: (err, stack) {
         // ignore: only_throw_errors
-        throw err;
+        throw err!;
```

Comment:
How about making it more null safe: `throw err ?? Exception('Non specified exception.');`

Reply:
I made it non-nullable instead, and used Error.throwWithStackTrace

This required Dart 2.16, so I've bumped the SDK constraints. Hope that's ok